packages/gestalt/src/Button.js

```diff
@@ -205,7 +213,7 @@ const ButtonWithForwardRef: React$AbstractComponent<unionProps, unionRefs> = for
         wrappedComponent="button"
       >
         {iconEnd ? (
-          <IconEnd text={buttonText} textColor={textColor} icon={iconEnd} size={size} />
+          <IconEnd text={buttonText} textColor="darkGray" icon={iconEnd} size={size} />
```

Comment:
I don't think these two should be hard coded, it creates issues when you disabled or select them
![Screen Shot 2021-05-14 at 9 48 35 AM](https://user-images.githubusercontent.com/5125094/118302895-a2d81c00-b499-11eb-9e38-88b7f9c4549d.png)

Was there a need to change them?

Reply:
LEGACY DEBUGGING... MY BAD

DONE